catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Registry.view.js

```diff
@@ -327,14 +330,24 @@ define([
                 if (this.model) {
                     data = this.model.toJSON();
                 }
-                var array = [];
+                var nodes = [];
                 _.each(this.options.nodes, function (node) {
-                    array.push({
+                    if(node.get('identityNode')){
+                        return;
+                    }
+                    nodes.push({
                         name: node.getObjectOfType('urn:registry:federation:node')[0].Name,
                         id: node.get('id')
                     });
                 });
-                data.registry = array;
+                nodes.sort(function (a, b) {
```

Comment:
you could use underscore like this:
`_.sortBy([{ key: "a" }, { key: "c" }, { key: "B" }], function (o) { return o.key.toLowerCase() })`